### ISSUE TITLE
Naruse specific modules in own branch/repo

### ISSUE THREAD

**Issue by AtjonTV** [ATVG-Studios/Naruse-V#3]
It may make sense to pull all Naruse specific modules into a separate branch or repository.

This would allow users of V to clone and use Naruse modules without the need of getting the whole Naruse-V package (Compiler + vlib).

When doing so, code must be changed and adopted to the V Std, as it differs from the Naruse-V Std.

**Comment by AtjonTV** [comment]
Naruse modules have been moved into the `naruse-modules` branch. The branch was then linked as submodule into the `nlib` directroy and linked into the `vlib` directory by using symlinks.

Development of new modules (like Git) are done in `naruse-module/xx`. Once initial development is done they must be merged into `naruse-modules`
